antarest/study/storage/variantstudy/model/command/create_renewables_cluster.py

```diff
@@ -49,9 +48,13 @@ def _apply_config(
             return (
                 CommandOutput(
                     status=False,
-                    message=f"enr_modelling must be {ENR_MODELLING.CLUSTERS.value}",
+                    message=(
+                        f"Parameter 'renewable-generation-modelling'"
+                        f" must be set to '{ENR_MODELLING.CLUSTERS.value}'"
+                        f" instead of '{study_data.enr_modelling}'"
+                    ),
                 ),
-                dict(),
+                {},
             )
```

Comment:
[optional]: we could be slightly more concise
```python
    def _apply_config(
        self, study_data: FileStudyTreeConfig
    ) -> Tuple[CommandOutput, Dict[str, Any]]:
        if study_data.enr_modelling != ENR_MODELLING.CLUSTERS.value:
            message = (
                f"Parameter 'renewable-generation-modelling' must be set to "
                f"'{ENR_MODELLING.CLUSTERS.value}' instead of '{study_data.enr_modelling}'"
            )
            return CommandOutput(status=False, message=message), {}
```

Reply:
agree